tests/foreman/ui/test_flatpak.py

```diff
@@ -36,3 +47,111 @@ def test_flatpak_remotes(target_sat, function_org, function_flatpak_remote):
         remote = remotes[0]
         assert remote['Name'] == function_flatpak_remote.remote.name
         assert remote['URL'] == function_flatpak_remote.remote.flatpak_index_url
+
+        details = session.flatpak_remotes.read_remote_details(name=remote['Name'])
+        assert details['title'] == remote['Name']
+        assert details['url'] == remote['URL']
+        match = re.search(r'of\s+(\d+)\s+items', details['pagination']['_items'])
```

Comment:
Non-blocking, I would maybe put some comment around here to make it clearer that we are searching for that digit and matching it in the assert, but your call on this.

Reply:
Ok, makes sense, I'll add it.